cdk/lib/stripe-patrons-data.ts

```diff
@@ -39,10 +43,10 @@ function parameterStorePolicy(scope: GuStack, appName: string) {
 }
 
 class StripePatronsDataLambda extends GuScheduledLambda {
-  constructor(scope: GuStack, id: string, appName: string) {
+  constructor(scope: GuStack, id: string, appName: string, buildNumber: string) {
     super(scope, id, {
       app: appName,
-      fileName: `${appName}.jar`,
+      fileName: `${buildNumber}.jar`,
```

Comment:
```suggestion
      fileName: `${appName}-${buildNumber}.jar`,
```

Reply:
👍 I've pushed these changes up in a single commit - the workflow file needed a change as well